discord-scripts/invite-management.ts

```diff
@@ -172,13 +172,13 @@ export default async function sendInvite(discordClient: Client, robot: Robot) {
         if (typeof clientName.value === "string") {
           await interaction.deferReply({ ephemeral: true })
           const normalizedClientName = clientName.value
-            .replace(/\s+/g, "-")
+            .replace(/[^a-zA-Z0-9]/g, "-")
             .toLowerCase()
-          const internalChannelName = `int-${normalizedClientName}-audit`
-          const externalChannelName = `ext-${normalizedClientName}-audit`
+          const internalChannelName = `🔒int-${normalizedClientName}-audit`
+          const externalChannelName = `🔒ext-${normalizedClientName}-audit`
```

Comment:
IMO we should let the existing automation handle this, unless it's not working for some reason.

Reply:
This was a weird one where it seemed to not work like it should of without this definition and 🔒 prefix.